packages/manager/cypress/component/features/firewalls/firewall-rule-table.spec.tsx

```diff
@@ -0,0 +1,154 @@
+import '@4tw/cypress-drag-drop';
```

Comment:
```suggestion
// Using this lib only for mouse drag-and-drop interactions
import '@4tw/cypress-drag-drop';
```

I think it would be helpful to add a comment here for clarity